runtime/libffi/CMakeLists.txt

```diff
@@ -40,7 +40,7 @@ target_include_directories(ffi PUBLIC ${CMAKE_CURRENT_SOURCE_DIR})
 
 #platform specific stuff hacked in
 if(OMR_ARCH_X86)
-	if(NOT OMR_ENV_DATA64)
+	if(OMR_ENV_DATA32 AND (NOT OMR_OS_WINDOWS))
```

Comment:
I think we should prefer using `OMR_ENV_DATA64` over `OMR_ENV_DATA32` for consistency with the rest of this file, so the condition should be `NOT (OMR_ENV_DATA64 OR OMR_OS_WINDOWS)`.